# We don't include it, so don't build the plugin (will fail otherwise)
# Mainlined with https://github.com/psi4/psi4/pull/436
# wpoely86@gmail.com
diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index 7752eed..ae27cc1 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -4,7 +4,9 @@ add_subdirectory(mointegrals)
 add_subdirectory(mollerplesset2)
 add_subdirectory(sointegrals)
 add_subdirectory(skeleton)
+if (Ambit_FOUND)
 add_subdirectory(skeletonambit)
+endif()
 add_subdirectory(skeletonaointegrals)
 add_subdirectory(skeletonmointegrals)
 add_subdirectory(skeletonsointegrals)
